### DISCUSSION TITLE
AlignedUMAP - reference papers

### DISCUSSION THREAD

**Discussion by mglowacki100** [lmcinnes/umap#966] [General]
Hi,
Aligned UMAP is very interesting approach and I want to dig a little deeper.
1. Are there any reference papers regarding AlignedUMAP method? I see that there is "Timecluster..." in scientific papers section, but it seems to me only related but not strictly about it.
2. It seems to me that alignment methods are independent from UMAP and they could be used with any dimensional reduction technique. Am I right? Could you point me to some research about it? I've tried to google it, but unfortunately I've got only 'false positive' results, so probably I don't know right keywords to find it.

**Comment by lmcinnes** [comment]
I did consider writing something up, but it is largely just "the obvious thing to do", so I didn't feel a paper from me would be very impactful. The ideas aren't really all that novel, and have been developed a number of times. Some that I know of are:

2-MAP: https://openaccess.thecvf.com/content_WACV_2020/html/Liu_2-MAP_Aligned_Visualizations_for_Comparison_of_High-Dimensional_Point_Sets_WACV_2020_paper.html
Dynamic t-SNE: https://qmro.qmul.ac.uk/xmlui/bitstream/handle/123456789/65040/Rauber%20Visualizing%20Time-Dependent%202016%20Accepted.pdf?sequence=2
Index t-SNE: https://arxiv.org/pdf/2109.10538.pdf

As far as I know these were all developed independently (and Aligned UMAP was developed independently as well; I have become aware of these later). They all end up with essentially the same solution, with varying minor differences, and Aligned UMAP is very similar as well.

Some of the things that are more unique to Aligned UMAP as a focus on support for partial relations, chaining relations, and aligning over a window of slices rather than just the previous / next. The other papers mentioned above have different focusses and spend more time and effort on other things.

At the very least this should provide you with a start on some references.

**Comment by mglowacki100** [comment]
Thanks @lmcinnes  once more. 
Btw. yesterday I watched  yours yt video "2023 02 22 Python Exchange - "A History of UMAP As a Python Open Source Project”"
https://www.youtube.com/watch?v=oyT12l0niPc&ab_channel=Don%27tUseThisCode-Training and 45:38 there is slide with various projects related/inspired by UMAP. I knew about couple of them mostly by arxiv sanity search, but there are a few new findings.
I think it'd be great to add "related project/approaches" in documentation, as UMAP page is a starter point for non-PCA dimensionality reduction for many folks.
